kitchensink/src/app/static/howToPages/DateTimePicker.jsx

```diff
@@ -0,0 +1,151 @@
+              timeZone="Europe/Zurich"
+              dateOnly={dateOnly}
+              selectStart
+            />
+          </td>
+          <td>
+            <DateTimePicker
+              label="To"
+              endDate={endDate}
+              startDate={startDate}
+              minDateTime={minDateTime}
+              maxDateTime={maxDateTime}
+              onChange={(value) => (value && setEndDate(new Date(value)))}
+              timeZone="Europe/Zurich"
+              dateOnly={dateOnly}
+              selectEnd
+            />
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  );
+  `;
+
+  return (
+    <>
+      <div className={styles.description}>
+        <p>
+          The DateTimePicker component allows you to select a single date or a range (i.e a From date and To date).
+        </p>
+        <p>
+          This section is to demonstrate how you would implement a From and To DateTimePicker coupling.
+        </p>
+      </div>
+      <div className={styles.topicContent}>
+        <Paragraph title="Getting started">
+          <DescriptionBox>
+            <div>
+              <ul>
+                <li>Import the DateTimePicker component.</li>
+                <li>Create your state hooks for storing the Start Date (From) and End Date (To).</li>
+                <li>selectStart and selectEnd are props that should be set. These activate the highlighting of the days when selecting the dates in the calendar dropdown.</li>
+              </ul>
+            </div>
+          </DescriptionBox>
+          <CodeBox>
+            {datepickercode}
+          </CodeBox>
+        </Paragraph>
+      </div>
+    </>
+  );
+};
+
+const DateUsage = ({
+  dateOnly,
+  minDateTime,
+  maxDateTime,
+  timeZone,
+}) => {
+
+  const [endDate, setEndDate] = useState();
+  const [startDate, setStartDate] = useState();
+
+  return (
+    <table>
+      <tbody>
+        <tr>
+          <td style={{ height: '200px' }}>
+            <DateTimePicker
+              label="From"
+              endDate={endDate}
+              startDate={startDate}
+              minDateTime={minDateTime}
+              maxDateTime={maxDateTime}
+              onChange={(value) => (value && setStartDate(new Date(value)))}
+              timeZone={timeZone}
+              dateOnly={dateOnly}
+              selectStart
+            />
+          </td>
+          <td>
+            <DateTimePicker
+              label="To"
+              endDate={endDate}
+              startDate={startDate}
+              minDateTime={minDateTime}
+              maxDateTime={maxDateTime}
+              onChange={(value) => (value && setEndDate(new Date(value)))}
+              timeZone={timeZone}
+              dateOnly={dateOnly}
+              selectEnd
+            />
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  );
+};
+
+DateUsage.defaultProps = {
+  dateOnly: false,
+  minDateTime: 0,
```

Comment:
Can we start with a bigger value here or use a text form to type the value? 🙂
It takes a lot of time to reach a realistic timestamp in Kitchensink by clicking the plus button 😄
![Screenshot 2021-11-09 at 13 29 38](https://user-images.githubusercontent.com/15834481/140924495-4e6cdf5e-ff95-48f6-b7e3-3365657a1857.png)

Reply:
From what @dparret said we will be going with no minimum and no maximum defaults.